tests/console/yast2_nis.pm

```diff
@@ -58,18 +57,16 @@ sub run() {
     type_string "/mounts_local";
     send_key 'alt-o';
     assert_screen 'nfs_server_added';                  # check Mount point
-    send_key 'alt-o';
-    assert_screen 'unable_to_mount_nfs';               # check error message and confirm with OK
-    send_key 'alt-o';
-    send_key 'alt-s';                                  # go back to nfs configuration and delete configuration created before
-    send_key 'alt-t';                                  # delete nfs client configuration
+    wait_screen_change { send_key 'alt-t' };           # go back to nfs configuration and delete configuration created before
     assert_screen 'nis_server_delete';                 # confirm to delete configuration
     send_key 'alt-y';
-    send_key 'alt-o';
+    wait_screen_change { send_key 'alt-o' };
     send_key 'alt-f';                                  # close the dialog...
     assert_screen 'nis_server_not_found';              # check error message for 'nis server not found'
     send_key 'alt-o';                                  # close it now even when config is not valid
+
 }
 1;
 
 # vim: set sw=4 et:
+
```

Comment:
empty line and line reference test: http://e13.suse.de/tests/1615
got removed now.